x-pack/plugins/security_solution/server/endpoint/services/actions/utils.test.ts

```diff
@@ -72,7 +72,7 @@ describe('When using Actions service utilities', () => {
         )
       ).toEqual({
         agents: ['6e6796b0-af39-4f12-b025-fcb06db499e5'],
-        command: 'unisolate',
+        command: 'kill-process',
```

Comment:
Not sure what's going on here @paul-tavares. I notice that the test has been [in this old PR](https://github.com/elastic/kibana/pull/134966/files#diff-d467296300e778b8a45a8b71f99ae87aa9ffebde04dded8f8a14c55705dcb267) as well. Looks like adding a new item in `RESPONSE_ACTION_API_COMMANDS_NAMES` should pick out a random action, but looks like it's moved from first item(isolate) to the second (unisolate) and now third (kill-process). 🤷

Reply:
I think I know why - our seeded random generator will always pick the same value for the same seed, but in this case, the array of values that it picks from changed, so I think it's expected that the result would now also change.